section/two.py

```diff
@@ -61,48 +66,31 @@ def section_2_3(self):
     def section_2_4(self):
         self.name = "2.4 Ensure CloudTrail trails are integrated with CloudWatch Logs"
         self.scored = True
-        self.passed = True
-
-        if self.CT.describe_trails():
-            resp = self.CT.describe_trails()
-
-            if u'CloudWatchLogsRoleArn' in resp["trailList"][0].keys():
-                CWname = resp["trailList"][0][u'Name']
-                CWstatus = self.CT.get_trail_status(Name=CWname)
-
-                if CWstatus[u'IsLogging']:
-                    LastDelivery = CWstatus[
-                        u'LatestCloudWatchLogsDeliveryTime']
-                    delta = datetime.now(tzlocal()) - LastDelivery
-                    if delta.days > 1:
-                        # logs are considered too old, integration test is
-                        # failed
-                        self.passed = False
-                else:
-                    # currently not Logging
-                    self.passed = False
-            else:  # no CloudWatch integration
-                self.passed = False
-
-        else:
-            # problem with describe_trails
-            self.passed = False
-
+        self.passed = False
+        if self.CT.trailsReport:
+            for trail in self.CT.trailsReport:
+                if 'CloudWatchLogsLogGroupArn' in trail:
+                    trailStatus = self.CT.getTrailStatus(trail["Name"])
+                    date = trailStatus["LatestCloudWatchLogsDeliveryTime"].replace(tzinfo=None)
+                    if date > datetime.now() - timedelta(days=1):
+                        self.passed = True
         print("{}, passed : {}".format(self.name, self.passed))
 
+    def section_2_5(self):
+        self.name = "2.5 Ensure AWS Config is enabled in all regions"
+        self.scored = True
+        self.passed = False
+        # This test requires the VM to be in US-East-1 (Virginia)
+        # It will be implemented later
+
     def section_2_6(self):
-        self.name = "Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket"
+        self.name = "2.6 Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket"
         self.scored = True
-        # new style : Spoonboy, i would rather have false by default
         self.passed = False
-        if self.CT.describe_trails():
-            resp = self.CT.describe_trails()
-            if (resp["trailList"][0]):
-                # we get the CloudTrail Bucket name
-                buck = resp["trailList"][0][u'S3BucketName']
-                # we check if attribute logging_enabled is not None
-                if self.S3.BucketLogging(buck).logging_enabled is not None:
-                    self.passed = True
```

Comment:
The name of the bucket were cloud trail send the log is already gathered in test 2.1 in order to avoid multiple function call in other test.
So the current test of 2.6 is a bit simplified